controller_manager/src/controller_manager.cpp

```diff
@@ -467,13 +467,21 @@ controller_interface::ControllerInterfaceBaseSharedPtr ControllerManager::load_c
       controller = chainable_loader_->createSharedInstance(controller_type);
     }
   }
-  catch (const pluginlib::CreateClassException & e)
+  catch (const std::exception & e)
   {
     RCLCPP_ERROR(
-      get_logger(), "Error happened during creation of controller '%s' with type '%s':\n%s",
+      get_logger(), "Caught exception while loading the controller '%s' of plugin type '%s':\n%s",
       controller_name.c_str(), controller_type.c_str(), e.what());
     return nullptr;
   }
+  catch (...)
+  {
+    RCLCPP_ERROR(
+      get_logger(),
+      "Caught unknown exception while loading the controller '%s' of plugin type '%s'",
+      controller_name.c_str(), controller_type.c_str());
+    return nullptr;
+  }
```

Comment:
the problem with catching everything this way is that we'll mask all the obscure weird exceptions people should be aware their code is throwing. I suggest we print this but then rethrow the same exception as whoever has non standard exceptions thrown around should be aware of it

here's an example of throwing potatoes: https://godbolt.org/z/vbe7GxGzM

Reply:
I love the example! hahahhaah

Reply:
Done!